srcpkgs/libnova/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'libnova'
+pkgname=libnova
+version=0.15.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="autoconf automake libtool"
+short_desc="A double precision astrometry and astrodynamics library"
+maintainer="travankor <travankor@tuta.io>"
+license="LGPL-2.0-only"
```

Comment:
LGPL-2.0-or-later

$ head -n 5 masterdir/builddir/libnova-0.15.0/src/airmass.c
/*
 *  This library is free software; you can redistribute it and/or
 *  modify it under the terms of the GNU Lesser General Public
 *  License as published by the Free Software Foundation; either
 *  version 2 of the License, or (at your option) any later version.